test/test_ci.py

```diff
@@ -54,8 +47,9 @@ def new_ec2_from_ami():
     then starts another EC-2 instance, based on the new AMI, then changes the password (which is expired),
     and finally  returns that EC-2 name together with the new temporary password.
     """
-    default_password = generate_random_password(12)
-    new_password = generate_random_password(12)
+    default_password = generate_random_password(length=12)
+    new_password = generate_random_password(length=14)
+    assert default_password != new_password
```

Comment:
maybe, reroll, instead of assert, because this fluke shouldn't be a reason for a failure

Reply:
actually, the assert can't happen, because both passwords differ in length. It's just a validation in case someone changes the logic in future.

Reply:
oh, that I didn't see, should we add a small comment